### ISSUE TITLE
[PD] Reflect on your plan to live as a developer

### ISSUE THREAD

**Issue by StanStarishko** [StanStarishko/My-Coursework-Planner#12]
### Coursework content
Now that you have your spreadsheet, you reviewed it, and you are confident it is a good picture of your availability and commitments on a typical week, we can reflect on it.

An important goal of this exercise is to observe your life from an external perspective. 
- Write a statement about your reflection. It should be at least 50 words long.
- Don't forget to check your text with a grammar tool, such as Grammarly

**Read it a few times aloud** and ensure you haven’t missed anything. Ask yourself if this is what you had planned or a realistic description of what happens. **Make any changes and iterations** to it. 

Now look at the number of hours you dedicate to each activity per week. 

Ensure you include your **energy and concentration levels** when studying or CYF training.

Read the updated sample and iterate again on it, ensuring it has the expected level of detail.

**Share your reflection with your pair.** 
Ask them to ask you questions about the **accuracy** of it. 
Get them to help you to **refine it**.

### Estimated time in hours

1

### What is the purpose of this assignment?
Observing your week and reflecting on your real availability will be key to ensure you have a clear plan to achieve your ultimate goal, which is a job as a developer. 

Be transparent in all what you are writing and thinking about, to ensure you have a reliable plan. Remember: the plan doesn't end when the CYF course ends.

### How to submit
Add the following as a comment on this issue:
- If you feel comfortable doing so, link to the updated Google Docs with your statement. Make sure anyone with this link can comment on it. 
- Add a comment about your discussion with your pair (up to 100 words). Remember to check the grammar and spelling before posting it.

**Comment by StanStarishko** [comment]
https://docs.google.com/spreadsheets/d/1EgYd9iUWwJbQU86fvI8u6GQud2lkfOhDleYYIzt1AUE/edit?usp=sharing

**Comment by StanStarishko** [comment]
>- Write a statement about your reflection. It should be at least 50 words long.
>- Don't forget to check your text with a grammar tool, such as Grammarly

I have long thought about how to develop my weaknesses and how to highlight and find them. Highlighting and finding is not a problem. But setting smart goals and achieving them is a small problem. I read the materials of this exercise and they seemed useful to me. Usually, I do not see the benefit in such materials and exercises.

**Comment by StanStarishko** [comment]
> ### How to submit
> Add the following as a comment on this issue:
> - If you feel comfortable doing so, link to the updated Google Docs with your statement. Make sure anyone with this link > can comment on it. 
> - Add a comment about your discussion with your pair (up to 100 words). Remember to check the grammar and spelling > before posting it. 

Discussing this document with my partner allowed me to make sure that I understood the exercise correctly. My wording and goals were clear and understandable enough. At the end of our discussion, I only had to fill in the "Self evaluation" column. I had not done this before discussing with my partner, in order to really evaluate the purity of understanding my wording by another person. So that the outside view would be as objective as possible. The result was expected, although I had to make a small clarification in the document on the second question that I am currently unemployed, for a clearer understanding by the other person
